src/options/theme.rs

```diff
@@ -126,19 +125,18 @@ fn should_detect_dark_light(opt: &cli::Opt) -> bool {
 }
 
 #[cfg(not(test))]
-fn detect_light_mode() -> bool {
+fn detect_light_mode() -> Option<bool> {
     use terminal_colorsaurus::{color_scheme, ColorScheme, QueryOptions};
-    color_scheme(QueryOptions::default()).unwrap_or_default() == ColorScheme::Light
+    color_scheme(QueryOptions::default())
```

Comment:
Because this PR has such a nicely thought out commit list/stack, could you also add a bonus commit to move this `use` up and then do whatever so the test cfg is happy? This non-test crate should not hide here :)